examples/datasets/load_limo_dataset.py

```diff
@@ -0,0 +1,173 @@
+"""
+========================================================
+Import single subject from LIMO data set into MNE-Python
+========================================================
+
+Here we a define function to extract the eeg signal data
+from the LIMO structures in the LIMO dataset, see [1]_ and:
+
+    https://datashare.is.ed.ac.uk/handle/10283/2189?show=full
+
+    https://github.com/LIMO-EEG-Toolbox
+
+The code allows to:
+
+    - Fetch single subjects epochs data for the LIMO data set.
```

Comment:
do not indent this, it will show up as a quoted list instead of standard list

Reply:
Ok, I hope the new version is better.